README.md

```diff
@@ -127,59 +127,26 @@ It's possible to package the repo's example application with `electron-builder`.
 
 ![theia-trace-extension](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-0.0.2.png)
 
 ### Open a trace
-To open a trace in the Theia Trace Extension, use the **File Explorer** in Theia to navigate to the trace directory. Then right-mouse click on the trace and select menu **Open With->Open Trace**.
+When opening a trace, if the selection is a single file, then the tool will open the file directly as a trace.
 
-![Open Trace](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-open-traces-0.0.2.png)
-
-If the selection is a single file, then the tool will open the file directly as a trace.
+The example Trace Compass trace server above supports LTTng Kernel and UST traces. Example LTTng traces can be retrieved from the [Trace Compass Tutorials](https://github.com/dorsal-lab/tracevizlab). Just download the archive [TraceCompassTutorialTraces](https://github.com/dorsal-lab/tracevizlab/blob/master/labs/TraceCompassTutorialTraces.tgz), extract them into a local directory on your computer. They can also be automatically downloaded by running `yarn download:sample-traces` from the repository's root.
 
+#### Create a trace group
 If the selection is a directory, then the tool will look for traces in **Common Trace Format (CTF)** format, such as **Linux Tracing Toolkit traces (LTTng)** Kernel and UST (Userspace) traces, and open all found CTF traces together under the same timeline. The trace events of each CTF trace will be analyzed in chronological order. With this you're able to open e.g. LTTng Kernel and UST Traces at the same time.
 
-The example Trace Compass trace server above supports LTTng Kernel and UST traces. Example LTTng traces can be retrieved from the [Trace Compass Tutorials](https://github.com/dorsal-lab/tracevizlab). Just download the archive [TraceCompassTutorialTraces](https://github.com/dorsal-lab/tracevizlab/blob/master/labs/TraceCompassTutorialTraces.tgz), extract them into a local directory on your computer. They can also be automatically downloaded by running `yarn download:sample-traces` from the repository's root.
-
 ### Open the Trace Viewer
 To open the **Trace Viewer**, select menu **View** from the top-level menu and then select **Trace Viewer** in the list of views. Then the **Trace Viewer** icon will be added on the left navbar, below the **File Explorer** Icon.
 
 Select the **Trace Viewer** icon to switch to the trace viewer.
 
 ![Trace Viewer](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-trace-explorer-0.0.2.png)
 
-Now, you will see 3 sections: **Opened Traces**, **Available Views** and **Time Graph Tooltip**. Select the trace that you opened and the available views will be populated. Now you can select different views and they will be added into a single container view for that trace.
-
-### Navigation and zooming
```

Comment:
why was this chapter removed? Instead is should be updated to reflect the what is currently implemented for zooming and navigation.

Reply:
Back when I submitted this I just removed it because it was outdated I didn't feel I had time to update it correctly, but now I feel I have the time!  

I will probably focus on the main methods for zooming and panning rather that being exhaustive.